src/wallet-account.ts

```diff
@@ -87,6 +87,11 @@ export class WalletConnection {
         window.location.assign(newUrl.toString());
     }
```

Comment:
While we're here, can we fix `requestSignIn` and mark `successUrl` and `failureUrl` as optional? In many of our examples we skip these arguments, but if you convert a project to typescript you get yelled at for not providing them.

Reply:
I put those two arguments as optional and ran `env NODE_ENV=ci yarn test` and everything passed.
Cleaned up the typedocs of the `requestSignIn` method, good eye.